internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -82,6 +84,7 @@ const (
 	MCS_CERT_NAME                       = "ca.crt"
 	OPENSHIFT_MACHINE_API_NAMESPACE     = "openshift-machine-api"
 	ASSISTED_DEPLOY_METHOD              = "start_assisted_install"
+	NODE_LABEL_PREFIX                   = "node-label."
```

Comment:
We should prefix it with "bmac.agent-install.openshift.io/" like the other annotations we added.

Reply:
The downside is that it uses more characters from the label key, but I don't see any other choice.

Reply:
Right, good point about the slash.  On the one hand, it should be clear that this annotation "belongs to" BMAC/agent-install; on the other hand, we need to conform to the label key limitations.  @mhrivnak, what do you think?  How about "bmac.agent-install.nodel-label."?

Reply:
Changed the prefix to bmac.agent-install.openshift.io.node-label